### ISSUE TITLE
Embycon constantly prompts to Select User

### ISSUE THREAD

**Issue by johanburati** [faush01/plugin.video.embycon#66]
I am hitting this bug https://github.com/faush01/plugin.video.embycon/issues/38, *Embycon* is constantly prompting me to select the user.

I have uninstalled *Embycon* and reinstalled it but still having the issue.
I am using LibreELEC (official) **9.1.501**, with Python **2.7.16** and Embycon **1.9.32**

When it happens I can see this warning in the log:
```
2019-09-27 03:45:51.465 T:1936878016  NOTICE: VideoPlayer: finished waiting
2019-09-27 03:52:25.258 T:1936878016 WARNING: RunScript called for a non-script addon 'plugin.video.embycon'. This behaviour is deprecated.
2019-09-27 03:52:36.539 T:1324147584  NOTICE: [plugin.video.youtube] Running: YouTube (6.5.1) on Leia (Kodi-18.4) with Python 2.7.16
```
Any idea of to fix this ?

**Comment by faush01** [comment]
It sounds like you have a corrupt addon settings file, you will probably need to delete your EmbyCon settings file.
It is located in the addon data dir in the user settings dir

https://kodi.wiki/view/Userdata

then

\userdata\addon_data\plugin.video.embycon\settings.xml

**Comment by johanburati** [comment]
I have reinstalled everything but I am still having the issue.

I am using **LibreELEC v9.1.501**,
I used **EmbyCon** with a older version of **LibreELEC** before and did not have this issue.

Could the extension need some update to work with this new release of **LibreELEC** ? 
The warning message seems to indicate that: 
`RunScript called for a non-script addon 'plugin.video.embycon'. This behaviour is deprecated.`
